### ISSUE TITLE
VZD ETL Bug

### ISSUE THREAD

**Issue by sergiogcx** [cityofaustin/atd-data-tech#2248]
Airflow reports this problem:

```
[2020-04-02 03:05:23,973] {{docker_operator.py:244}} INFO - Traceback (most recent call last):
  File "/app/process_hasura_import.py", line 26, in <module>
    from process.helpers_import import *
  File "/app/process/helpers_import.py", line 20, in <module>
    from .helpers_import_fields import CRIS_TXDOT_FIELDS, \
ImportError: cannot import name 'CRIS_TXDOT_COMPARE_FIELD_TYPE' from 'process.helpers_import_fields' (/app/process/helpers_import_fields.py)
[2020-04-02 03:05:24,195] {{taskinstance.py:1047}} ERROR - docker container failed: {'Error': None, 'StatusCode': 1}
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 922, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/docker_operator.py", line 248, in execute
    raise AirflowException('docker container failed: ' + repr(result))
airflow.exceptions.AirflowException: docker container failed: {'Error': None, 'StatusCode': 1}
```

**Comment by sergiogcx** [comment]
PR:
https://github.com/cityofaustin/atd-vz-data/pull/731

The problem stopped the ETL process last night, but it did not fully break the pipeline.

I was able to replicate the error locally, which means that I lost that code sometime around the time when I was reverting and fixing other branches, or probably around conflict resolutions.

Added again the missing code from and old local branch, and everything is back in order locally.

Merging to master, re-running.
